streams/az.m3u

```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
 https://edge1.socialsmart.tv/aznews/smil/aznews/720p/chunks.m3u8
 #EXTINF:-1 tvg-id="AznewsTV.az",AZ News (720p)
 https://swow1.socialsmart.tv/aznews/smil:aznews.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AzTV.az",Az TV (1080p)
-https://str.yodacdn.net/aztv/index.m3u8
 #EXTINF:-1 tvg-id="BakuTV.az",Baku TV (720p)
 https://rtmp.baku.tv/live/bakutv_720p.m3u8
-#EXTINF:-1 tvg-id="CBC.az",CBC (720p)
-http://cbctvlive.flashmediacast.com:1935/CBCTVLive/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="CBC.az",CBC (720p) [Geo-blocked]
+http://api.canlitvplayer.com/str.yodacdn.net/cbc/mono.m3u8?token=37c73919564746534a6a85ed9337ae487a29cd5f-6ff7674e65450e3c6650c43aa4442988-1664284043-1664273243
+#EXTINF:-1 tvg-id="CBCSport.az",CBC SPORT
+https://mn-nl.mncdn.com/cbcsports_live/cbcsports/chunklist.m3u8
 #EXTINF:-1 tvg-id="ELTV.az",EL TV (260p)
 http://85.132.53.162:1935/live/eltv/playlist.m3u8
-#EXTINF:-1 tvg-id="IctimaiTV.az",Ictimai TV
-http://109.205.166.68/server124/ictimai_tv/index.m3u8
-#EXTINF:-1 tvg-id="IdmanTV.az",İdman TV (576p)
-http://109.205.166.68/server124/idman_az/index.m3u8
 #EXTINF:-1 tvg-id="KanalTuran.fr",Kanal Turan (576p)
 https://donvolga.com/Turan-TV/index.m3u8
 #EXTINF:-1 tvg-id="KepezTV.az",Kəpəz TV (540p) [Not 24/7]
 http://85.132.81.184:8080/arbkepez/live/index.m3u8
-#EXTINF:-1 tvg-id="MedeniyyetTV.az",Medeniyyet TV (406p)
-https://str.yodacdn.net/medeniyyet/index.m3u8
 #EXTINF:-1 tvg-id="MuganTV.az",Mugan TV (1080p) [Not 24/7]
 https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
 #EXTINF:-1 tvg-id="QafqazTV.az",Qafqaz TV [Geo-blocked]
 https://str1.yodacdn.net/qafkaz/playlist.m3u8
 #EXTINF:-1 tvg-id="RealTV.az",Real TV [Geo-blocked]
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:-1 tvg-id="AlZahraTV.ir",AL-ZAHRA TV
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AgroTV.tr",AGRO TV AZERBAIJAN [Geo-blocked]
```

Comment:
To move with the "tr.m3u" playlist, to correspond with the tvg-id.

Reply:
Indeed, but we classify the channel by their country of origin, not by the intended countries they broadcast to.

It is a channel based in Turkey, so it should be classified in the tr.m3u playlist. It would still appear on the Azerbaijan playlist (after CI publication) as it is made for their diaspora.